generator/templates/backend/ProcTable.cpp

```diff
@@ -83,7 +84,11 @@ namespace {{namespace}} {
                     {% for arg in method.arguments %}
                         {% set cppType = as_cppType(arg.type.name) %}
                         {% if arg.type.category in ["enum", "bitmask"] %}
-                            if (!IsValid{{cppType}}(static_cast<nxt::{{cppType}}>({{as_varName(arg.name)}}))) error = true;
+                            MaybeError {{as_varName(arg.name)}}Valid = Validate{{cppType}}(static_cast<nxt::{{cppType}}>({{as_varName(arg.name)}}));
```

Comment:
Nit: Is there any way to refactor the {{as_varName(arg.name)}} out of this? It occurs five times.

Reply:
Done, you can do it with a control block that looks like `{% set argName = as_varName(arg.name)%}`